v2/ticker_service.go

```diff
@@ -118,9 +125,14 @@ func (s *ListPriceChangeStatsService) Do(ctx context.Context, opts ...RequestOpt
 		method:   http.MethodGet,
 		endpoint: "/api/v3/ticker/24hr",
 	}
+
 	if s.symbol != nil {
 		r.setParam("symbol", *s.symbol)
+	} else if s.symbols != nil {
+		s, _ := json.Marshal(s.symbols)
+		r.setParam("symbols", string(s))
```

Comment:
`setParam` already supports `[]string` now, so you do need to manually `Marshal` it.
```suggestion
		r.setParam("symbols", s)
```

Reply:
thanks